recipes/dhcp/dhcp4.inc

```diff
@@ -56,38 +56,45 @@ do_install_extra_files () {
 	install -m 0644 ${SRCDIR}/default-server \
 		${D}${sysconfdir}/default/dhcp-server
 
-	rm -f ${D}${sysconfdir}/dhclient.conf
-	rm -f ${D}${sysconfdir}/dhcpd.conf
-	install -m 0644 ${SRCDIR}/dhclient.conf \
-		${D}${sysconfdir}/dhcp/dhclient.conf
-	install -m 0644 ${SRCDIR}/dhcpd.conf \
-		${D}${sysconfdir}/dhcp/dhcpd.conf
-
+	mv ${D}${sysconfdir}/dhcpd.conf.example \
+	   	${D}${sysconfdir}/dhcp/dhcpd.conf.example
+	mv ${D}${sysconfdir}/dhclient.conf.example \
+	   	${D}${sysconfdir}/dhcp/dhclient.conf.example
 	install -d ${D}${base_sbindir}/
```

Comment:
This is done to preserve how the package was placing the files before my changes.